ddl/ddl_tiflash_test.go

```diff
@@ -38,7 +38,8 @@ import (
 	"github.com/pingcap/tidb/store/gcworker"
 	"github.com/pingcap/tidb/store/mockstore"
 	"github.com/pingcap/tidb/store/mockstore/unistore"
-	"github.com/pingcap/tidb/util/testkit"
+	testkit "github.com/pingcap/tidb/testkit"
```

Comment:
```suggestion
	"github.com/pingcap/tidb/testkit"
```

@CalvinNeo you don't resolve the problem actually :P

Reply:
Have not pushed latest commits...sorry